code/game/turfs/simulated/walls_misc.dm

```diff
@@ -1,6 +1,6 @@
 /turf/simulated/wall/cult
-	name = "wall"
-	desc = "The patterns engraved on the wall seem to shift as you try to focus on them. You feel sick."
+	name = "runed wall"
+	desc = "A wall made of a strange alloy. It's shockingly cold to the touch."
```

Comment:
I prefer the old text.
eldritch text and patterns that can't be discerned by the mind are great.